wd_core.au3

```diff
@@ -233,17 +235,14 @@ Func _WD_CreateSession($sCapabilities = Default)
 
 		If @error Then
 			Local $sMessage = Json_Get($oJSON, "[value][message]")
-
-			Return SetError(__WD_Error($sFuncName, $_WD_ERROR_Exception, $sMessage), 0, "")
+			$iErr = $_WD_ERROR_JSON
```

Comment:
should we change error form `$_WD_ERROR_Exception` to `$_WD_ERROR_JSON` after all issue related to `Json_Get($oJSON` ?

Reply:
No, and I disagree with your change above. Not all errors will be JSON related. For example --

```
__WD_Post ==> Invalid argument [5] : HTTP status = 400 ResponseText={"value":{"error":"invalid argument","message":"acceptInsecureCerts is not boolean: \"xxx\"","stacktrace":""}}
_WD_CreateSession ==> Webdriver Exception [10]
```

I haven't tested it, but I was thinking about doing something like this --

```
		If @error Then
			$sMessage = Json_Get($oJSON, "[value][message]")
			__WD_DetectError($iErr, $oJSON)
```

Reply:
as so far: [revert _WD_ERROR_Exception in _WD_CreateSession()](https://github.com/Danp2/au3WebDriver/pull/321/commits/8cc11505dcd11166dac0bdd5e346290eaeaf6f5b)